build_forklift_bazel.sh

```diff
@@ -21,24 +24,57 @@ export REGISTRY_ACCOUNT=""
 export CONTAINER_CMD=$(which docker)
 
 # REGISTRY_ACCOUNT cannot be empty with docker
-REGISTRY_ACCOUNT=ci make push-openstack-populator-image
-REGISTRY_ACCOUNT=ci make push-ovirt-populator-image
+if [ "${PROVIDER_NAME}" = "openstack" ]; then
+    REGISTRY_ACCOUNT=ci make push-openstack-populator-image
+fi
+if [ "${PROVIDER_NAME}" = "ovirt" ]; then
+    REGISTRY_ACCOUNT=ci make push-ovirt-populator-image
+fi
+
 REGISTRY_ACCOUNT=ci make push-populator-controller-image
 
 bazel run push-forklift-api
 bazel run push-forklift-controller
 bazel run push-forklift-validation
 bazel run push-forklift-operator
+
+ACTION_ENV="--action_env CONTROLLER_IMAGE=${REGISTRY}/forklift-controller:${REGISTRY_TAG} \
+    --action_env VALIDATION_IMAGE=${REGISTRY}/forklift-validation:${REGISTRY_TAG} \
+    --action_env OPERATOR_IMAGE=${REGISTRY}/forklift-operator:${REGISTRY_TAG}
```

Comment:
Missing a slash here `\`

Reply:
+1
@bennyz asked me to review this PR last week, I did - and my only comment was on this line (`don't we need \ at the end of this line?`) so I was so surprised to see that you also comments on this but then I realized that my comment was in "Pending" state.....